### DISCUSSION TITLE
Flag to avoid providing certs while testing

### DISCUSSION THREAD

**Discussion by gskumar1010** [i-am-bee/beeai-framework#222] [Q&A]
Hi,
I am trying to check if there is a flag or a way to avoid providing below values 

IBM_VLLM_ROOT_CERT, IBM_VLLM_CERT_CHAIN, IBM_VLLM_PRIVATE_KEY
in https://github.com/i-am-bee/bee-agent-framework/blob/main/.env.template#L31

Thanks

**Comment by Tomas2D** [comment]
You do not need them. Just remove them from your `.env` and tests will be passed.

**Comment by gskumar1010** [comment]
Hmm, I run into below error when I dont set IBM_VLLM_ROOT_CERT, IBM_VLLM_CERT_CHAIN, IBM_VLLM_PRIVATE_KEY in my .env

```
 yarn start examples/llms/providers/ibm-vllm.ts

node:internal/modules/run_main:122
    triggerUncaughtException(
    ^

[FrameworkError: Required variable 'IBM_VLLM_ROOT_CERT' is not set!] {
  isFatal: false,
  isRetryable: true,
  context: {},
  [errors]: []
}
```
Thanks

**Reply by Tomas2D** [reply]
You need to pass such ENVs to run IBM vLLM. If you don't have access to such an environment, use a different provider.
